packages/nc-gui/components/cell/Text.vue

```diff
@@ -27,6 +27,11 @@ const focus: VNodeRef = (el) => (el as HTMLInputElement)?.focus()
     class="h-full w-full outline-none bg-transparent"
     :class="{ '!px-2': editEnabled }"
     @blur="editEnabled = false"
+    @keydown.down.stop
+    @keydown.left.stop
+    @keydown.right.stop
+    @keydown.up.stop
+    @keydown.delete.stop
```

Comment:
Are these all necessary?

Reply:
Yes all are required otherwise it will switch between cells which is not required when a cell(typable) is in editable state

Reply:
Ok :)